### DISCUSSION TITLE
Error: selecting output plugin '': no such plugin

### DISCUSSION THREAD

**Discussion by Satherad** [cmus/cmus#1101] [Troubleshooting]
---

1) `uname -a`:
```
Linux localhost.localdomain 4.18.0-240.el8.x86_64 #1 SMP Fri Sep 25 19:48:47 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux
```

2) `cmus --version`:
```
cmus v2.9.1
```

3) `cmus --plugins`:
```
Input Plugins: /usr/local/lib/cmus/ip
  cue:
    Priority: 50
    File Types:
    MIME Types: application/x-cue
  wav:
    Priority: 50
    File Types: wav
    MIME Types:

Output Plugins: /usr/local/lib/cmus/op
  pulse
  oss

```
Hello everyone,
When I launched cmus, the error as in the title showed up. I installed pulseaudio dev package and changed plugin output to pulse but it didn't work. Is there anyway to fix it?I am working on Centos.

**Comment by pgaskin** [comment]
Try temporarily renaming your cmus configuration directory and see if it starts correctly.

**Reply by Satherad** [reply]
I am new to this. Turned out solution was very simple. I had to switch to another user to make it work. Now everything is fine. Going back to the previous directory works as well. But thx anyway for trying to help and quick response!
